### PR TITLE
Add service_name and package_name parameters

### LINKED ISSUE #16: Support alternative package & service name for Kibana
Hi, 

Firstly, thanks so much for finally coming out with an "Elastic" Kibana module! I'd love to start making use of the module, but can't because I rebuild Kibana from source to add additional features and name the package (& service) differently in order to de-conflict with the official release. 

Could I please request that you support alternative names for the package and service resources? 

Many thanks, 
Nick G

### PR DESCRIPTION
**Pull request by phaedriel**
#### Pull Request (PR) description

- Add 'service_name' parameter (default kibana)
- Add 'package_name' parameter (default kibana)

#### This Pull Request (PR) fixes the following issues

Fixes #16

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -33,6 +34,7 @@
   Boolean $manage_repo,
   Boolean $oss,
   Optional[String] $package_source,
+  String[1] $service_name,
```
Comment (smortex) [inline manifests/init.pp:37]: Please set the default value here instead of Hiera so that the default value is properly documented.

**Review by smortex — COMMENTED** [review]
One minor issue to fix before merging, thanks!

File: manifests/init.pp (hunk 1 above)
Comment (phaedriel) [reply manifests/init.pp:37]: Ok but i will do this modification when i come back after my holidays ;) perhaps i will add something to manage package name in same time

Comment (phaedriel) [reply manifests/init.pp]: Done and i add a new parameter 'package_name'

**Review by tuxmea — APPROVED** [review]

**Review by smortex — APPROVED** [review]

**Pull request merged**